pymeasure/display/windows/managed_window.py

```diff
@@ -155,6 +163,15 @@ def __init__(self,
         self._layout()
 
     def _setup_ui(self):
+        if self.filename_input:
+            self.writefile_toggle = QtWidgets.QCheckBox('Write file', self)
+            self.writefile_toggle.setLayoutDirection(QtCore.Qt.RightToLeft)
+            self.writefile_toggle.setChecked(True)
+            self.writefile_toggle.stateChanged.connect(self.toggle_file_dir_input_active)
```

Comment:
I like tooltips, which describe, what an option does. Maybe you could add one.
I guess this toggle switches between writing a file into the directory and a temporary file, doesn`t it?